input/examples/practitioner-guthridge-jarred.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Compliant with Services Australia Test Data -->
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="guthridge-jarred"/>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-name">
+        <valueString value="Practitioner - Dr. Jarred Guthridge"/>
+    </extension>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
+        <valueMarkdown value="Dr. Jarred Guthridge"/>
+    </extension>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-practitioner"/>
+    </meta>
+    <identifier>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="NPI"/>
+                <display value="National provider identifier"/>
+            </coding>
+            <text value="HPI-I"/>
+        </type>
+        <system value="http://ns.electronichealth.net.au/id/hi/hpii/1.0"/>
+        <value value="8003611566718288"/>
+    </identifier>
+    <identifier>
```

Comment:
Remove Prescriber identifier - it isn't a Must Support identifier type.

Reply:
Done